packages/dropdownable/stories/Dropdownable.stories.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import { withReadme } from "storybook-readme";
+import { withInfo } from "@storybook/addon-info";
+
```

Comment:
I would love to see some knobs especially for preferred directions and open in place to make the demo more interactive.